packages/@webex/plugin-cc/src/features/Agentconfig.ts

```diff
@@ -1,8 +1,9 @@
 import {IAgentProfile, WORK_TYPE_CODE} from './types';
```

Comment:
Is this correct that the agentConfig file saves the agent config and any helper methods which acts on the agentConfig ?